helpers/lib/common.js

```diff
@@ -10,21 +10,21 @@ function isValidURL(val) {
 }
 
 /*
- * Based on https://github.com/jonschlinkert/get-value/blob/3.0.1/index.js, but
+ * Based on https://github.com/jonschlinkert/get-value/blob/2.0.6/index.js, but
  * with configurability that was not used in handlebars-helpers removed.
  */
 function getValue(object, path) {
```

Comment:
Won't be it better to unwrap path here? What if someone uses this utility function with SafeString?

Reply:
Or maybe we can indicate that it doesn't support SafeString ( throw an error)

Reply:
I've moved the unwrap call into `getValue` :)